linera-rpc/src/config.rs

```diff
@@ -140,6 +140,10 @@ pub struct ValidatorInternalNetworkPreConfig<P> {
     pub host: String,
     /// The port the proxy listens on the internal network.
     pub port: u16,
+    /// The server configurations for the linera-exporter.
+    /// They can be used as optional locations to forward notifications to destinations other than
+    /// the proxy, by the workers.
+    pub block_exporters: Vec<ExporterServiceConfig>,
```

Comment:
For backcompatibility, I believe this should be at the end of the struct marked as optional for serde.